Makefile

```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
 	@yarn run test-ci
 	@echo ""
 
-test-python:
-	@echo "--> Running Python tests"
-	# This gets called by getsentry
-	pytest tests/integration tests/sentry
-
 test-python-ci:
-	make build-platform-assets
```

Comment:
Is this a related change to this PR? If not, would it make sense to separate into a different PR?

I'm trying `python setup.py bdist_wheel --build-number 0`, where is that executed? And how does it call `from sentry.utils.integrationdocs import sync_docs; sync_docs()`?

Reply:
No, not related, just a small cleanup. It's so small, I think it's fine to just toss into this PR.

Reply:
```
docker/builder.sh
15:python setup.py bdist_wheel --build-number 0
```

It runs this stuff in setup.py:

```
class SentryBuildCommand(BuildCommand):
    def run(self):
        from distutils import log as distutils_log

        distutils_log.set_threshold(distutils_log.WARN)

        if not IS_LIGHT_BUILD:
            self.run_command("build_integration_docs")
            self.run_command("build_assets")
            self.run_command("build_js_sdk_registry")
        BuildCommand.run(self)
```

That's the `run_command("build_integration_docs")`.